#!/usr/bin/env python
PACKAGE = "ros_recognizer"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("icp_max_iter",        int_t,   0, "ICP max iterations (default 5)", 5, 1, 101)
gen.add("icp_corr_dist",    double_t,   0, "ICP correspondence distance (default .005)", .01, 0, .1)
gen.add("icp_two_pass",       bool_t,   0, "Use IterativeClosestPoint",  True)

exit(gen.generate(PACKAGE, "pcl_recognizer", "Refiner"))